src/containers/VerifyUser/VerifyUser.js

```diff
@@ -0,0 +1,107 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Button } from 'react-materialize';
+import { withRouter } from 'react-router-dom';
+import './VerifyUser.scss';
+import { connect } from 'react-redux';
+import { verifyUser } from '../../requests/VerifyUserRequests';
+
+/**
+ * @class VerifyUser
+ * @extends {Component}
+ */
+class VerifyUser extends Component {
+  /**
+   * @constructor function
+   * @param {*} props React properties
+   */
+  constructor() {
+    super();
+    this.state = { success: false, loading: true };
+    this.redirectToHome = this.redirectToHome.bind(this);
+  }
+
+  /**
+   * @description - This method runs whenever redux state changes
+   * @returns {object} state
+   * @param {object} props
+   * @memberof VerifyUser
+   */
+  static getDerivedStateFromProps(props) {
+    return { success: props.success, loading: props.loading };
+  }
+
+  /**
+   * @description - This method runs whenever redux state changes
+   * @returns {object} state
+   * @param {object} props
+   * @param {object} state
+   * @memberof VerifyUser
+   */
+  componentDidMount() {
+    this.props.verifyUser(this.props.match.params.verificationToken);
+  }
+
+  /**
+   * @memberOf handleredirection to home page
+   * @method closeModal
+   * @param {*} event
+   * @return {*} boolean
+   */
+  redirectToHome() {
+    this.props.history.push('/');
+  }
+
+  /**
+   *
+   * @description - This method renders the jsx for this component
+   * @returns {jsx} - jsx
+   * @memberof VerifyUser
+   */
+  render() {
+    const { loading, success } = this.state;
+    return (
+      <div>
+        { loading && <h5>Loading...</h5>}
+        {
+          !loading
+          && <div className="row verifyDiv">
+            <div className="col m8 offset-m2">
+              {
+                success
+                && <div>
```

Comment:
you can extract this block into a component and then pass the message as props... 
Since the only difference seems to be the success message

Reply:
Sounds good to me. Thanks!